Nautilus/Documentation/guides/assetbundles.md

```diff
@@ -0,0 +1,114 @@
+> 1. Locate where your editor is. 
+> You can do that by opening Unity Hub, going to Installs, and then right clicking on the version in question and clicking "Show in Explorer"
+> 2. Go to ``Editor\Data\Resources\PackageManager\ProjectTemplates``. You will find a ``manifest.json`` and ``UnityLicense.json`` files. delete them both.
+> 3. Kill Unity Hub in your task manager and open it again.
+> 
+> You should now be able to create your project!
+
+The Unity editor will now open. This may take a while.  
+Once the editor is open, drag your model from your file manager into the Scene window.
+
+You can either click on your model in the scene window or in the hierarchy on the left to select it.
+
+Next, click on ``Window`` at the top of the Unity editor, and then click ``Package Manager``.  
+Click on the dropdown at the top left and select ``Unity Registry`` if not on it already.  
+Search for "Asset Bundle Browser" and install it.
+
+Next, drag your model from the hierarchy on the left into the ``Assets`` section at the bottom.  
+This will create what is called a ``Prefab``. It can be instantiated (created/spawned) multiple times, but they are all copies of the original.
+
+To edit the prefab, double click on it in the ``Assets`` window.  
+You can add components to the prefab by clicking "Add Component" in the inspector on the right.  
+One example of a component is a ``RigidBody``, which can apply physics to your Prefab.  
+Any components you add will be included in the AssetBundle.
+
+You <font color="red">**should not**</font>, however, add scripts you made in the editor, as the AssetBundle will not link them to the scripts in your modding project and you will get errors.
+
+After all your prefab configuration is done, click on it in the ``Assets`` window, and click on "Asset Bundle" in the bottom right of the Inspector.  
+Here you can either assign your prefab to an existing AssetBundle or create a new one.  
+![assetbundles-assigning](../images/guides/assetbundles-assigning.png)
+
+Once you've assigned your AssetBundle, click ``Window``, and then ``AssetBundleBrowser``.  
+Here you can see all the assets that will be put into your AssetBundle.  
+![assetbundles-browser](../images/guides/assetbundles-browser.png)
+
+Click on the build tab, and change the ``Build Target`` to "Standalone Windows" if it isn't already.  
+You can also configure the ``Output Path`` of the build.  
+After all that is set, click ``Build``.
+
+Once your AssetBundle is built, move the built file to the location of your mod in the Plugins folder.  
+I prefer to put them inside a folder namded "Assets", but that isn't necessary.  
+![assetbundles-assets-folder](../images/guides/assetbundles-assets-folder.png)
+
+Now the hard part is done! All you need to do is load it into your mod's code.  
+To do this, you will need to add an extra reference to your mod.  
+Right click ``References`` in Visual Studio under your project and select add reference.
+
```

Comment:
Again I'd suggest headers for all the individual parts of the process, like maybe not everyone needs to know how to add this reference. Could mention that this isn't necessary for Nautilus mod template users at the top of the section.